cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -140,7 +140,7 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
           |as well as the ability to find these nodes, to a role `regularUsers` you can use the following `GRANT MATCH` query.""".stripMargin)
 
       query("GRANT MATCH { language, length } ON GRAPH neo4j NODES Message TO regularUsers", ResultAssertions((r) => {
```

Comment:
Maybe update the description for `MATCH` to make it clear that it is not split into `READ` and `TRAVERSE` but is its own thing